src/Workspaces/Core/Portable/FindSymbols/FindReferences/DependentProjectsFinder.cs

```diff
@@ -364,6 +364,11 @@ private static Lazy<HashSet<string>> CreateInternalsVisibleToMap(IAssemblySymbol
                     }
 
                     var value = (string)typeNameConstant.Value;
+                    if (string.IsNullOrWhiteSpace(value))
```

Comment:
Er, misspoke. If the string *didn't* contain the comma, then we left the whitespace in. This can't matter, right?!